### ISSUE TITLE
Chef LDAP config breaks permissions

### ISSUE THREAD

**Issue by svucich** [chef/chef-server#1058]
I had a working chef server. I decided to then enable LDAP, linked my existing management/admin user and now I can no longer add clients to the chef server.
I thought just disabling the LDAP config would resolve the issue (remove LDAP data from chef-server.rb and run reconfigure commands), but it doesn't.
Here is where I am currently at (Also, the user, according to the gui has all the required permissions, in the admin group with create etc...)

`knife bootstrap windows winrm machinename.mydomain.local -x 'mydomain.local\svucich' -P password -N machinename
Creating new client for aklsvucich01
Creating new node for aklsvucich01
ERROR: You authenticated successfully to https://chef.mydomain.local/organizations/mycompany as sysadmin but you are not authorized for this action
Response:  missing create permission`

I have tested this on 2 previously working chef servers and end up with the same result. A system that I can't add any clients to rendering it near on useless :(

Chef Server version 12.10.0

Any assistance in sorting this is appreciated.

**Comment by svucich** [comment]
Further this, I have been digging using the local (to the chef serer) cli commands and found that all my previous LDAP users are still showing up in the cli but not the GUI???

`sudo chef-server-ctl user-list
pivotal
ross_taylor
sysadmin
user_0
user_1`

sysadmin should be the only user left in the org that I care about - Also should be an admin.

Then I run the list-server-admins command to see what shows up there:

`sudo chef-server-ctl list-server-admins
pivotal
sysadmin`

So we can see that the user 'sysadmin' is on the server, is enabled for the admin group, but is still unable to create nodes/clients via bootstrap w/or w/out LDAP enabled - But only after LDAP has been enabled once.

**Comment by svucich** [comment]
I have also just tried the following:

`sudo chef-server-ctl user-create sv firstname lastname email@domain 'P@55word!'`

Then

`sudo chef-server-ctl grant-server-admin-permissions sv`

Which now shows my user as a server admin, but gives me the same result as before.

`Response: missing create permission`

**Comment by svucich** [comment]
I have also reached out to the chef community to try and get some assistance. [Here ](https://discourse.chef.io/t/ldap-breaks-create-permissions/10292/3)is my post on discourse.chef.io

**Comment by svucich** [comment]
OK - I have resolved the issue. It seems that there are hidden ACL's (to avoid over complicating the GUI apparently).

I installed the ACL Ruby GEM via the following command:

chef gem install knife-acl
Then, I need to add the required 'create' permissions to each of the acl containers, example as follows:

knife acl add group admins containers clients create
knife acl add group admins containers nodes create
This resolved my immediate issue with bootstrapping a node and also allows me to add the permissions with a working LDAP enabled on the chef-manage server where the LDAP breaks the existing ACL's.

Credit to the work done [here ](https://github.com/chef/knife-acl)which gave me all I needed.
